### DISCUSSION TITLE
对 skynet.profile `diff_time` 逻辑有些疑问

### DISCUSSION THREAD

**Discussion by t0350** [cloudwu/skynet#1699] [Q&A] [answered]
请问下 https://github.com/cloudwu/skynet/blob/master/service-src/service_snlua.c#L127 `diff_time` 这里逻辑，什么情况下 `now < start` ，以及为什么小于 0 的处理逻辑先加 0x10000 再作差？
谢谢!

**Comment by cloudwu** [comment, marked as answer]
正常不会发生。万一发生了, +0x10000 容易暴露问题。等发现了再看为什么。
